src/app/events/event-organize/event-organize.html

```
@@ -76,15 +76,34 @@ <h3 class="title-medium">
-                  <button
-                    mat-stroked-button
-                    (click)="cancelRegistration(user)"
-                    [disabled]="
-                      user.checkedIn || cancelRegistrationMutation.isPending()
-                    "
-                  >
-                    Cancel registration
-                  </button>
+                  <div class="flex flex-wrap gap-2">
+                    <button
+                      mat-stroked-button
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)